website/docs/operator/collection-configuration.md

```diff
@@ -0,0 +1,8 @@
+---
+id: collection-configuration
+title: Collection Configuration
+description: Provides documentation for Operators to configure a collection 
+using the Cumulus dashboard.
+---
+
+This document in under construction.

```

Comment:
Did you want to pull the information from the https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus#collection-configuration page and migrate to here?

Reply:
Will leave stubbed out, put in a card for content on this page, and work with Operators to develop content.